robowflex_library/src/io/visualization.cpp

```diff
@@ -377,6 +377,11 @@ void IO::RVIZHelper::removeMarker(const std::string &name)
         it->second.action = visualization_msgs::Marker::DELETE;
 }
 
+void IO::RVIZHelper::addMarker(const visualization_msgs::Marker &marker, const std::string &name)
```

Comment:
There are many other addMarker functions - they should all call this one for purposes of code reuse. I believe they all end up creating a visualization_msgs::Marker in the end.